src/index.html

```diff
@@ -15,8 +15,62 @@
       rel="stylesheet"
       href="./style.css"
     />
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:ital,wght@0,100;0,300;0,400;0,500;0,700;0,900;1,100;1,300;1,400;1,500;1,700;1,900&display=swap"
+      rel="stylesheet"
+    />
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <header>
+      <div class="div-img">
+        <a>
+          <img
+            src="images/logo.png"
+            alt="logo"
+          />
+        </a>
+      </div>
+
+      <nav>
+        <ul class="nav-links">
```

Comment:
The class name 'nav-links' doesn't give a clear understanding of what this ul contains or represents. It would be better to name it according to its purpose, for example 'nav__list'.